src/main/resources/io/jenkins/plugins/explain_error/ConsolePageDecorator/footer.jelly

```diff
@@ -24,6 +25,12 @@
               <l:spinner text="Analyzing error logs..."/>
             </div>
             <pre id="explain-error-content" class="jenkins-!-margin-bottom-0 ${hasExplanation?'':'jenkins-hidden'}">${hasExplanation ? existingExplanation.explanation : ''}</pre>
+            <div class="jenkins-!-margin-top-3">
+              <a id="explain-error-url" href="${hasExplanation ? existingExplanation.urlString: ''}" class="build-status-pill ${hasExplanation?'':'jenkins-hidden'}" target="_self" >
+                <l:icon src="symbol-status-red" class="icon-md" />
```

Comment:
I would avoid using the build status icon here. Better to use something from ionicons-api plugin
Maybe `symbol-warning-outline plugin-ionicons-api`  or `symbol-alert-circle-outline plugin-ionicons-api` (need to add a dependency to that plugin then)
With the button class above you can remove the class attribute.

Reply:
@mawinter69  done

Reply:
<img width="1056" height="477" alt="image" src="https://github.com/user-attachments/assets/f50401cb-5496-46a5-8ca7-3fad41878058" />